routes/tag.go

```diff
@@ -26,6 +26,9 @@ func (r *tagHandler) Get(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, gin.H{"Error": "Bad Sorting Option"})
 		return
 	}
+	if args.Sorting == "text" {
+		args.Sorting = "tag_content"
+	}
```

Comment:
If there are multiple sort settings like sort=tag_content,updated_at this wouldn't work, right?